demo/ubuntu-demo-opae/test_fpga.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh -e
+
+# Test FPGA by running 2 OPAE samples: nlb3 and nlb0
+# nlb3 is expected to succeed, nlb0 - to fail
+green () { echo "\033[0;32m$1\033[0m";}
```

Comment:
I guess the flag `-e` is missing here

Reply:
it was there before. I removed it as it messed up 'kubectl logs <pod>' output.